src/components/Button/index.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import CreateCustomMaterialUlComponent from "utils/CreateCustomMaterialUlComponent";
+import ButtonMaterial from "@material-ui/core/Button";
+
+const styles = {
```

Comment:
Tu też dwie szkoły - wcześniej byłem zwolennikiem trzymania styli w jednym pliku z komponentem, ale teraz wywalam do styles.js(ts) nawet jak jest jeden styled component. tu tez mozesz dac to do styles.js, a index.js zmien na index.jsx, bo ten plik zawiera jsx

No i dobrze sie trzymac jednej konwencji na caly projekt, czyli jak robisz wszedzie styled component, to w jednym miejscu nie rob nagle styles={styles}. Chyba ze chciales sprobowac innych metod :)
Takie proste rzeczy jak buttony, linki, typografia, dobrze zrobic w theme materialowym. wtedy masz gotowy swoj domyslny theme na caly projekt i nie musisz juz nigdzie nic pisac, jezeli nie ma zadnych dziwnych styli nagle

Reply:
"a index.js zmien na index.jsx, bo ten plik zawiera jsx" - przecież react ma już wbudowany kompilator jsx, wytłumacz mi proszę dokładnie dlaczego powinienem tylko w tym jednym pliku użyć jsx a w innych już nie.

During...